ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -1676,6 +1676,39 @@ def test_torch_pow(
     )
 
 
+@handle_frontend_test(
+    fn_tree="torch.float_power",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Also, we want to test for all valid dtypes (`get_dtypes("valid")`). If the function only supports certain dtypes, you should define this using  `@with_unsupported_dtypes` or `@with_supported_dtypes`.